### ISSUE TITLE
Venu 3

### ISSUE THREAD

**Issue by vicb** [blotspot/garmin-watchface-protomolecule#9]
Thanks for this awesome watch face.

Do you think it would be possible to add support for the Venu 3?

Thanks!

**Comment by vicb** [comment]
I am trying to update the project to support the Venu3 but I can not even run it for venu2.

I have the following error:

```
Executing task: java -Xms1g -Dfile.encoding=UTF-8 -Dapple.awt.UIElement=true -jar /home/victor/.Garmin/ConnectIQ/Sdks/connectiq-sdk-lin-7.0.2.beta-2024-01-10-acc16aead/bin/monkeybrains.jar -o bin/garminwatchfaceprotomolecule.prg -f /home/victor/code/garmin-watchface-protomolecule/monkey.jungle -y /home/victor/connectiq-sdk-manager-linux/developer_key -d venu2_sim -w 

ERROR: venu2: /home/victor/code/garmin-watchface-protomolecule/source/Settings.mc:89,4: Cannot assign value '$.Toybox.Lang.Array[$.Toybox.Lang.ResourceId, $.Toybox.Lang.ResourceId, $.Toybox.Lang.ResourceId, $.Toybox.Lang.ResourceId, $.Toybox.Lang.ResourceId, $.Toybox.Lang.ResourceId, $.Toybox.Lang.ResourceId, $.Toybox.Lang.ResourceId, $.Toybox.Lang.ResourceId, $.Toybox.Lang.ResourceId, $.Toybox.Lang.ResourceId, $.Toybox.Lang.ResourceId, $.Toybox.Lang.ResourceId, $.Toybox.Lang.ResourceId]' to member ':DataFieldRez'.
ERROR: venu2: /home/victor/code/garmin-watchface-protomolecule/source/datafield/DateAndTime.mc:19,4: Cannot assign value '$.Toybox.Lang.Array[$.Toybox.Lang.ResourceId, $.Toybox.Lang.ResourceId, $.Toybox.Lang.ResourceId, $.Toybox.Lang.ResourceId, $.Toybox.Lang.ResourceId, $.Toybox.Lang.ResourceId, $.Toybox.Lang.ResourceId, $.Toybox.Lang.ResourceId, $.Toybox.Lang.ResourceId, $.Toybox.Lang.ResourceId, $.Toybox.Lang.ResourceId, $.Toybox.Lang.ResourceId]' to member ':Months'.
ERROR: venu2: /home/victor/code/garmin-watchface-protomolecule/source/datafield/DateAndTime.mc:34,4: Cannot assign value '$.Toybox.Lang.Array[$.Toybox.Lang.ResourceId, $.Toybox.Lang.ResourceId, $.Toybox.Lang.ResourceId, $.Toybox.Lang.ResourceId, $.Toybox.Lang.ResourceId, $.Toybox.Lang.ResourceId, $.Toybox.Lang.ResourceId]' to member ':DayOfWeek'.
ERROR: venu2: /home/victor/code/garmin-watchface-protomolecule/source/settingsMenu/ProtomoleculeSettingsMenu.mc:235,4: Cannot assign value '$.Toybox.Lang.Array[$.Toybox.Lang.ResourceId, $.Toybox.Lang.ResourceId, $.Toybox.Lang.ResourceId, $.Toybox.Lang.ResourceId]' to member ':_theme'.
ERROR: venu2: /home/victor/code/garmin-watchface-protomolecule/source/settingsMenu/ProtomoleculeSettingsMenu.mc:249,4: Cannot assign value '$.Toybox.Lang.Array[$.Toybox.Lang.ResourceId, $.Toybox.Lang.ResourceId]' to member ':_layout'.
```

Compiling the sample watch faces from the SDK works fine.

Any hint on what could go wrong here?

**Comment by blotspot** [comment]
Sorry for the late reply. ~~I fixed that error. It must have been something that came through a later SDK. Initializing the resources array with `Array<Symbol>` worked at one point, but then they changed it to `Array<Toybox.Lang.ResourceId>`. Compiling it with an earlier SDK should have fixed your issue.~~ Don't mind me. I'm just a bit confused.

Thanks for your effort and the PR, I really appreciate it. Even though late, I did just update the App in the store, and it should now support the Venu 3
